From 38f5c60819b9ce216faaed06f2b60b54f8349087 Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Sat, 29 Oct 2016 13:38:34 +0200
Subject: [PATCH] Install the bash completion files in to a proper location

/etc is for user configuration, not configuration shipped with a package.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index eadd571..e355a49 100644
--- a/Makefile
+++ b/Makefile
@@ -13,7 +13,7 @@ DIST_DIR = $(MINIPRO)-$(VERSION)
 BIN_DIR=$(DESTDIR)/usr/bin/
 UDEV_RULES_DIR=$(DESTDIR)/usr/lib/udev/rules.d/
 MAN_DIR=$(DESTDIR)/usr/share/man/man1/
-COMPLETIONS_DIR=$(DESTDIR)/etc/bash_completion.d/
+COMPLETIONS_DIR=$(DESTDIR)/usr/share/bash_completion.d/
 
 libusb_CFLAGS = `pkg-config --cflags libusb-1.0`
 libusb_LIBS = `pkg-config --libs libusb-1.0`
